de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/DhlShipperGatewayClient.java

```diff
@@ -367,11 +369,49 @@ private CreateShipmentOrderRequest createDHLShipmentOrderRequest(
 					shipmentOrderTypeShipment.setShipper(shipperType);
 				}
 
+				//				{
```

Comment:
again, please clean this up, or add a comment explaining the use of this commented out code